### ISSUE TITLE
Raw legumes & dried legumes recipe inconsistency

### ISSUE THREAD

**Issue by Tharn** [CleverRaven/Cataclysm-DDA#32916]
# Describe the bug

1x raw beans turns into 6x dried beans.
1x raw lentils turns into 2x dried lentils.

However, all recipes I've checked will use the same amount of dried & raw legumes. Meaning drying first, then cooking, will result in a 600% more efficient use of beans and a 200% more efficient use of lentils. This doesn't make sense. Recommend changing these recipes to either 

- Make 1 raw legume in = 1 dried legume out. Fix harvest yield if required. Possibly shorten crafting times. See that volume & weight still makes sense.
- Change all food recipes to use 6x the amount of dried beans / 2x the amount of dried lentils compared to their raw counterparts. This strikes me as less ideal.

# Steps To Reproduce

1. Get you some legumes.
2. Use the drying recipes.
3. Compare amounts of food you're able to cook.

# Expected behavior

Kitchen crafting should not magically create a surplus of food. There is no reason raw legumes should be less efficient.

# Versions and configuration

 - OS: Win 8.1 x64
 - Game Version: 9424
 - Graphics version: Tiles
 - Mods loaded: None that matter

**Comment by TechyBen** [comment]
Size wise, lentils are small, right? So the number of them, seems right. Is it just the volume/weight/kcal that needs correcting?

**Comment by KorGgenT** [comment]
just to clarify, due to nutrients and calories being inherited there is no real surplus of food.

**Comment by Tharn** [comment]
Just checked it.

Cooking a Deluxe Veggie Beans & Rice turns both raw & cooked beans into `cooked beans (1)`, meaning you do get 6x the amount because the `dried beans` recipe gives you 6 portions.

On top of that, `raw beans` show as having 56kcal per unit while `dried beans` give 203kcal. Going by the resulting portions, shouldn't it be closer to the other way around?

**Comment by int-ua** [comment]
Please verify if it's still relevant.

**Comment by OliveChukar** [comment]
Tested in ver 0.E-9111-gd1e702b-dirty. Drying beans and lentils now gives one dried bean/lentil for every raw bean/lentil. Used debug to learn all recipes then looked up all foods that use beans/lentils and they all seem to use the same amounts. Cooked meat soup with beans/lentils and dry or raw the only difference in output was that the lentil soup showed what it was made with but the bean soup all said that it was made with cooked beans. 

When I made dried beans/lentils both showed greater calorie content then the input. I do not know if this is a different issue.
